### ISSUE TITLE
Assets filenames: hash after query

### ISSUE THREAD

**Issue by dkrnl** [iampava/imagemin-webp-webpack-plugin#7]
Hi!
Thx for plugin!

In my webpack.config.js used hash in query string:
```
{
    test: /\.(jpeg|jpg|png|gif|svg)$/i,
    loader: 'file-loader',
    options: { name: '[folder]/[name].[ext]?[hash]' }
},
```
But `imagemin-webp-webpack-plugin` ignore assets with hash, for example: `img/og-image.jpg?772a51fc9d55e34a5ede7f7d4ab8d37e`

Test example:
```
/\.(jpe?g|png)$/.test('img/og-image.jpg?772a51fc9d55e34a5ede7f7d4ab8d37e') // false
/\.(jpe?g|png)$/.test('img/og-image.jpg') // true
```
Need parsing file name?

**Comment by iampava** [comment]
Hey,

Changed the default RegExp to allow hashes after the extension. Haven't published the new version yet, since there were some other big chages related to this https://github.com/iampava/imagemin-webp-webpack-plugin/issues/6, but I did a push on GitHub and would very much appreciate if you could try it and tell me if it works.

Commit: https://github.com/iampava/imagemin-webp-webpack-plugin/commit/7409e6f16af32f3e0517a00e6d53c4b738240e21

PS: to try the latest code just replace the contents directly in node_modules

**Comment by iampava** [comment]
Published v3.0.0 with the fix. 

https://github.com/iampava/imagemin-webp-webpack-plugin/releases/tag/v3.0.0
